src/diffusers/schedulers/scheduling_euler_discrete.py

```diff
@@ -65,6 +65,8 @@ class EulerDiscreteScheduler(SchedulerMixin, ConfigMixin):
             `linear` or `scaled_linear`.
         trained_betas (`np.ndarray`, optional):
             option to pass an array of betas directly to the constructor to bypass `beta_start`, `beta_end` etc.
+        predict_epsilon (`bool`):
```

Comment:
Do we need this for the scheduler?

Reply:
I think so, the UNet predicts the denoised output in this case. I'll double check to see if we can do it differently somehow.